ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -2,59 +2,78 @@
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.AdministrationPage;
 import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
 import org.openmrs.reference.page.HeaderPage;
 import org.openmrs.reference.page.HomePage;
+import org.openmrs.reference.page.HtmlFormsPage;
 import org.openmrs.reference.page.ManageFormsPage;
+import org.openmrs.reference.page.ManageHtmlFormsPage;
 import org.openmrs.uitestframework.test.TestBase;
+import org.openqa.selenium.By;
 
 import static org.junit.Assert.assertNotNull;
 
-
 /**
  * Created by nata on 24.06.15.
  */
 public class AddFormTest extends TestBase {
+    private static String name = "newFormTest1";
+    private static String description = "description of new form";
+    private static String version = "1.2";
     private HomePage homePage;
     private HeaderPage headerPage;
     private ManageFormsPage manageForm;
+    private AdministrationPage administrationPage;
     private ClinicianFacingPatientDashboardPage patientDashboardPage;
+    private ManageHtmlFormsPage manageHtmlFormsPage;
+    private HtmlFormsPage htmlFormsPage;
 
     @Before
     public void setUp() throws Exception {
-        homePage = new HomePage(page);
-        assertPage(homePage);
-        headerPage = new HeaderPage(driver);
-        manageForm = new ManageFormsPage(driver);
-        patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
+	homePage = new HomePage(page);
+	assertPage(homePage.waitForPage());
+	administrationPage = new AdministrationPage(page);
+	headerPage = new HeaderPage(driver);
+	htmlFormsPage = new HtmlFormsPage(page);
+	manageHtmlFormsPage = new ManageHtmlFormsPage(page);
+	manageForm = new ManageFormsPage(driver);
+	patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
     }
 
-    @Ignore //ignore due to moving forms functionality
     @Test
+    @Category(BuildTests.class)
     public void addFormTest() throws Exception {
-        homePage.goToManageForm();
-        if (!manageForm.addPresent()) {
-            manageForm.delete();
-        }
-        manageForm.add();
-        manageForm.addLabel("Eye Report");
-        manageForm.addIcon("icon-align-justify");
-        manageForm.formIdFromUrl();
-        manageForm.save();
-        headerPage.clickOnHomeIcon();
-        homePage.goToActiveVisitPatient();
-        assertNotNull("Eye Report", patientDashboardPage.FORM_EXIST);
-        headerPage.clickOnHomeIcon();
-        homePage.goToManageForm();
-        manageForm.deletePath();
-
+	manageHtmlFormsPage = homePage.goToAdministration().clickOnManageHtmlForms();
+	if (manageHtmlFormsPage.getElementsIfExisting(By.xpath("//*[contains(text(), '" + name + "')]")).isEmpty()) {
+	    manageHtmlFormsPage.clickOnNewHtmlForm();
+	    htmlFormsPage.createNewFormTest(name, description, version);
```

Comment:
this has changed the indention @sherrif10

Reply:
@k-joseph  Are you suggesting we change the indentation to something else because the whole commit is now using the indentation format and size

Reply:
@k-joseph  Have resolved the remaining indentation